lib/settings/pages/general_settings_page.dart

```diff
@@ -720,6 +728,14 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
                           iconDisabled: Icons.link_rounded,
                           onToggle: (bool value) => setPreferences(LocalSettings.openLinksInExternalBrowser, value),
                         ),
+                        if (Platform.isIOS)
+                          ToggleOption(
+                            description: LocalSettings.openLinksInReaderMode.label,
+                            value: openInReaderMode,
+                            iconEnabled: Icons.menu_book_rounded,
+                            iconDisabled: Icons.menu_book_rounded,
```

Comment:
Looks like there's no "off" variant of this icon. I know we had been pretty good about finding one of each for most/all of our settings so far. Oh well!